src/Mod/Part/parttests/regression_tests.py

```diff
@@ -20,6 +20,81 @@ def test_issue_4456(self):
         with self.assertRaises(IndexError):
             result.pop()
 
+    def test_issue_15735(self):
+        """
+        15735: Point in sketch as loft profile won't work in dev, but works in stable
+        The following test is a simplified version of the issue, but the outcome is the same
+        """
+
+        # Arrange
+        from FreeCAD import Base
+        import Sketcher
+
+        self.Doc = newDocument("TestLoftWithPointSketch")
```

Comment:
Should there be a setUp method for this class that makes a document?  And a tearDown to remove it?  And if you want to be cute, a closeDocument control attribute like in TestTopologicalNaming Problem at the end?

Reply:
I agree, looks more coherent with other test files!